system-x/services/flink/src/main/java/software/tnb/flink/resource/local/FlinkContainers.java

```diff
@@ -0,0 +1,14 @@
+package software.tnb.flink.resource.local;
+
+import org.testcontainers.containers.DockerComposeContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+
+import java.io.File;
+
+public class FlinkContainers extends DockerComposeContainer<FlinkContainers> {
```

Comment:
I am not entirley sure about using DockerCompose, @dolearci you can tajke a look at the kafka system-x, in order to start kafka two docker images are needed, zookeeper and kafka itself, docker compose is not used, but the LocalKafka class takes care of starting and stopping both containers, can you try to follow this strategy?

wdyt @avano ?

Reply:
yes, that is something I would advise as well

anyway, having a service where you need to provide a docker compose file isn't very reusable

Reply:
@Croway I was trying the same approach as in kafka system-x, but it was not working. The jobs on flink server were scheduled but not running. Thats why i used docker compose variant and this is working successfully.

Reply:
mmm that is interesting, I smell like network issue, could it be that the 2 containers were running in different networks, and could not communicate with each other?

Reply:
I even tried creating a network and assigning it to each container, but that didn't work either. I tried manually running containers locally and that didn't work either. Somehow I managed to get it to run successfully using only the docker-compose.yml. The docker-compose file can be found here https://gitlab.cee.redhat.com/jboss-fuse-qe/t-n-b/tnb-tests/-/merge_requests/672#8b49b01b435bde3775d124d5766c3ee62824cd6b, if you have some free time we can look at it together, I just could not get it to work with two separate containers.

Reply:
I created separate containers for flink to make it work, can you check it again?